src/app/common/services/task-service.coffee

```diff
@@ -231,7 +231,7 @@ angular.module("doubtfire.common.services.tasks", [])
       action: "You should discuss this with your tutor and/or the convenor."
     time_exceeded:
       detail: "Time limit exceeded"
-      reason: "This work was submitted after the deadline, having missed the target and due dates."
+      reason: "This work was submitted after the deadline, having missed both the target and deadline."
```

Comment:
Lets add "date" to target here - so "target date" rather than target

Reply:
👍